docs/feature_rgblight.md

```diff
@@ -79,28 +79,40 @@ Your RGB lighting can be configured by placing these `#define`s in your `config.
 
 ## Animations
 
-Not only can this lighting be whatever color you want, if `RGBLIGHT_ANIMATIONS` is defined, you also have a number of animation modes at your disposal:
-
-|Mode |Description          |
-|-----|---------------------|
-|1    |Solid color          |
-|2-5  |Solid color breathing|
-|6-8  |Cycling rainbow      |
-|9-14 |Swirling rainbow     |
-|15-20|Snake                |
-|21-23|Knight               |
-|24   |Christmas            |
-|25-34|Static gradient      |
-|35   |RGB Test             |
-|36   |Alternating          |
+
+Not only can this lighting be whatever color you want,
+if `RGBLIGHT_EFFECT_xxxx` or `RGBLIGHT_ANIMATIONS` is defined, you also have a number of animation modes at your disposal:
+
+|Mode                                   |Description                            |
+|---------------------------------------|---------------------------------------|
+|`RGBLIGHT_MODE_STATIC_LIGHT`           |Solid color (this mode is always enabled) |
+|`RGBLIGHT_MODE_BREATHING + 0-3`        |Solid color breathing                  |
+|`RGBLIGHT_MODE_RAINBOW_MOOD + 0-2`     |Cycling rainbow                        |
+|`RGBLIGHT_MODE_RAINBOW_SWIRL + 0-5`    |Swirling rainbow                       |
+|`RGBLIGHT_MODE_SNAKE + 0-5`            |Snake                                  |
+|`RGBLIGHT_MODE_KNIGHT + 0-2`           |Knight                                 |
+|`RGBLIGHT_MODE_CHRISTMAS`              |Christmas                              |
+|`RGBLIGHT_MODE_STATIC_GRADIENT + 0-9`  |Static gradient                        |
+|`RGBLIGHT_MODE_RGB_TEST`               |RGB Test                               |
+|`RGBLIGHT_MODE_ALTERNATING`            |Alternating                            |
```

Comment:
I love having names for these. The numbers seem confusing however. Could we rearrange this a bit? I'm thinking something like this:

```
| Mode | Number of Speeds | Description |
|------|------------------|-------------|
| `RGBLIGHT_MODE_STATIC_LIGHT` | 1 | Solid color (this mode is always enabled) |
| `RGBLIGHT_MODE_BREATHING` | 3 | Solid color breathing
``` 

If you have another format that achieves the same goal I'm open to that too.

Reply:
Thank you.
I tried to change it. What do you think about this change.